Resources/Prototypes/Maps/Pools/default.yml

```
@@ -5,19 +5,19 @@
   - Bagel
   - Box
   - Cluster
-  #- Cog # Goobstation - cog is broken
+  #- Cog Goobstation - cog is broken
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)